WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40383,918 +40424,371 @@ e_latin_empire = {
 	}
 }
 
-d_miaphysite = {
```

Comment:
The one which has been discontinued? But I thought it was older: must be because of the old graphics you use.